package.json

```diff
@@ -9,6 +9,7 @@
   },
   "scripts": {
     "start": "NODE_ENV=development webpack-dev-server --mode development --hot",
+    "start:browserstack": "yarn start --host 0.0.0.0 --disable-host-check",
```

Comment:
Why not adding this flags in the start?

https://webpack.js.org/configuration/dev-server/#devserverdisablehostcheck

I see the `THIS IS NOT RECOMMENDED` warning, but I understand this is if you serve your app for production using the `webpack-dev-server`, which looks like a weird thing to do in the first place (normally you build the static files and use a regular HTTP server)

Is there anything im missing?

Reply:
this looks like a security concern, please don't turn it on by default
does it make you exposed when in develop?

Reply:
https://www.browserstack.com/question/759

iOS10+ restrictions force browserstack to push to a new URL (`bs-local`) so testing localhosted content wasn't working for iOS only.

didn't add it to `start` as i guess we don't technically need it there, only for browserstack usage - felt this was a bit safer

Reply:
But why is this a security concern? In case you are in a public network and someone access dex-react in your IP?
If I understand what it does, I don't see it very worrying, and it would help development when we need to test locally from our phones